CMakeLists.txt

```diff
@@ -59,4 +59,14 @@ set_target_properties(mycpplib PROPERTIES
 target_link_libraries(mycpplib
   ${catkin_LIBRARIES}
   ${Boost_LIBRARIES}
+
+)
+
+install(TARGETS mycpplib
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
 )
+
+file(GLOB py_files "scripts/*.py")
+install(PROGRAMS ${py_files}
```

Comment:
Please use the `catkin_install_python` macro to install python scripts. See http://docs.ros.org/kinetic/api/catkin/html/howto/format2/installing_python.html